ansible/tasks/setup-postgres.yml

```diff
@@ -1,127 +1,35 @@
-      - g++-10
+    name: postgresql-common
+    state: latest
+    update_cache: yes
 
-- name: Switch to GCC 10
+- name: Do not create main cluster
   shell:
-    cmd: update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-10 100 --slave /usr/bin/g++ g++ /usr/bin/g++-10 --slave /usr/bin/gcov gcov /usr/bin/gcov-10
+    cmd: sed -ri 's/#(create_main_cluster) .*$/\1 = false/' /etc/postgresql-common/createcluster.conf
 
-# Setup permissions
-- name: Update permissions for /var/tmp directory
-  file:
-    path: /var/tmp/
-    owner: root
-    group: root
-    mode: '1777'
-  become: yes
+- name: Postgres - install server
+  apt:
+    name: postgresql-{{ postgresql_major }}={{ postgresql_release }}-1.pgdg20.04+1
 
-# Building Postgres from source
-- name: Postgres - download latest release
-  get_url:
-    url: https://ftp.postgresql.org/pub/source/v{{ postgresql_release }}/postgresql-{{ postgresql_release }}.tar.gz
-    dest: /tmp
-    checksum: "{{ postgresql_release_checksum }}"
-    timeout: 60
-
-- name: Postgres - unpack archive
-  unarchive:
-    remote_src: yes
-    src: /tmp/postgresql-{{ postgresql_release }}.tar.gz
-    dest: /tmp
-
-- name: Setting CFLAGS (arm)
-  set_fact:
-    cflags: "-moutline-atomics -mtune=native -march=native -mcpu=native -fsigned-char -O2"
-  when: platform == "arm64" and ansible_distribution_version != "18.04"
-
-- name: Setting CFLAGS (arm) - Ubuntu 18.04
-  set_fact:
-    cflags: "-moutline-atomics -fsigned-char -O2"
-  when: platform == "arm64" and ansible_distribution_version == "18.04"
-
-- name: Setting CFLAGS (x86)
-  set_fact:
-    cflags: "-fsigned-char"
-  when: platform == "amd64"
-
-- name: Setting LLVM_CONFIG
-  set_fact:
-    llvm_config: "/usr/bin/llvm-config-11"
-    clang: "/usr/bin/clang-11"
-  when: ansible_distribution_version != "18.04"
-
-- name: Setting LLVM_CONFIG - Ubuntu 18.04
-  set_fact:
-    llvm_config: "/usr/bin/llvm-config-12"
-    clang: "/usr/bin/clang-12"
-  when: ansible_distribution_version == "18.04"
-
-- name: Postgres - configure
+- name: Create symlink to /usr/lib/postgresql/bin
   shell:
-    cmd: CFLAGS='{{ cflags }}' LLVM_CONFIG='{{ llvm_config }}' CLANG='{{ clang }}' ./configure --with-llvm --with-openssl --with-systemd --with-uuid=e2fs --with-libxml --with-icu --with-lz4 --exec-prefix=/usr/lib/postgresql --datarootdir=/var/lib/postgresql
-    chdir: /tmp/postgresql-{{ postgresql_release }}
-
-- name: Postgres - build
-  community.general.make:
-    target: world-bin
-    chdir: /tmp/postgresql-{{ postgresql_release }}
-    jobs: "{{ parallel_jobs | default(omit) }}"
-
-- name: Postgres - install
-  make:
-    target: install-world-bin
-    chdir: /tmp/postgresql-{{ postgresql_release }}
-
-- name: Create postgres group
-  group:
-    name: postgres
-    state: present
-
-# Create postgres user
-- name: Create postgres user
-  user:
-    name: postgres
-    shell: /bin/false
-    comment: Postgres user
-    group: postgres
-    groups: postgres,ssl-cert
```

Comment:
Should we ensure the newly created postgres user is also part of the `ssl-cert` group?

Reply:
I just checked that the default apt installation adds postgres user to `ssl-cert` group.

```bash
$ groups postgres
postgres : postgres ssl-cert
```